docs/topics/native/native-cocoapods.md

```diff
@@ -14,16 +14,117 @@ as well as [a Kotlin Gradle project and an Xcode project](native-cocoapods-xcode
 
 ## Set up an environment to work with CocoaPods
 
-Install the [CocoaPods dependency manager](https://cocoapods.org/):
+Install the [CocoaPods dependency manager](https://cocoapods.org/) using the installation tool of your choice:
+
+> If your device has an Apple M chip, follow our instructions [below](#apple-m-chips).
+>
+{type="note"}
+
+<tabs>
+<tab title="Homebrew">
+
+1. Install [Homebrew](https://brew.sh/) in case you don't have yet.
+2. Install Ruby. You can choose a specific version:
+
+    ```bash
+    brew install ruby@3.0
```

Comment:
My intuition tells me that instructions for installing CocoaPods with new instance of ruby on m1 and Intel should be roughly the same.
I don't have Intel mac near me, so I can't test this intuition, but macOS doesn't work differently on these chips, so the installation may be different only for the case when we are trying to install CocoaPods with system ruby
With M1 mac it is impossible to properly install the CocoaPods without changing the system ruby (somehow it works on one of my virtual machines, but currently I don't know why and I can't propose such a way to users)